tests/src/tests/defuse/accounts/auth_by_predecessor_id.rs

```diff
@@ -19,19 +20,29 @@ use crate::{
 
 #[tokio::test]
 #[rstest]
-async fn test(random_bytes: Vec<u8>) {
+async fn auth_by_predecessor_id(random_bytes: Vec<u8>) {
     let mut u = Unstructured::new(&random_bytes);
-    let env = Env::new().await;
+    let mut env = Env::new().await;
+
+    let ft = env.create_token().await;
+    let user = env.get_or_create_user().await;
```

Comment:
Test implementations shouldn't know whether these users were created before (for migration testing) or not, so maybe better to avoid confusion here and rename to `Env::create_user()`?
```suggestion
    let user = env.create_user().await;
```